includes/woopay/class-woopay-utilities.php

```diff
@@ -123,10 +122,10 @@ public function should_save_platform_customer() {
 		$session_data = [];
 
 		if ( isset( WC()->session ) && method_exists( WC()->session, 'has_session' ) && WC()->session->has_session() ) {
-			$session_data = WC()->session->get( WooPay_Extension::WOOPAY_SESSION_KEY );
+			$session_data = WC()->session->get( WooPay_Session::WOOPAY_SESSION_KEY );
 		}
 
-		return ( isset( $_POST['save_user_in_woopay'] ) && filter_var( wp_unslash( $_POST['save_user_in_woopay'] ), FILTER_VALIDATE_BOOLEAN ) ) || ( isset( $session_data['save_user_in_woopay'] ) && filter_var( $session_data['save_user_in_woopay'], FILTER_VALIDATE_BOOLEAN ) ); // phpcs:ignore WordPress.Security.NonceVerification
+		return isset( $session_data['save_user_in_woopay'] ) && filter_var( $session_data['save_user_in_woopay'], FILTER_VALIDATE_BOOLEAN );
```

Comment:
Why we're ignoring $_POST now?

Reply:
Using the ajax call we are always saving in session, so there's no need for the `$_POST`.